R/create_protocol.R

```diff
@@ -1,24 +1,52 @@
+#' fieldwork protocol
 #'
-#' @description This function will create a new folder based on values that are passed on via the parameters and creates a R-markdown (bookdown) skeleton based on a template file to start working on a new protocol.
+#' @description This function will create a new folder based on values that are
+#' passed on via the parameters and creates a R-markdown (bookdown) skeleton
+#' based on a template file to start working on a new protocol.
 #'
-#' @details The created folder is a new subfolder beneath `src` and contains a bookdown project. It is assumed that the src folder is a subfolder of an RStudio project with git version control.
+#' @details The created folder is a new subfolder beneath `src` and contains a
+#' bookdown project.
+#' It is assumed that the src folder is a subfolder of an RStudio project with
+#' git version control.
 #'
 #' @param title A character string giving the main title of the protocol
 #' @param subtitle A character string for an optional subtitle
-#' @param short_title A character string of less than 20 characters to use in folder and filenames
-#' @param authors A character string for authors of the form First name Last name and multiple authors separated by a comma
+#' @param short_title A character string of less than 20 characters to use in
+#' folder and filenames
+#' @param authors A character string for authors of the form First name Last
+#' name and multiple authors separated by a comma
 #' @param date A character string of the date in ISO 8601 format (YYYY-MM-DD)
-#' @param reviewers A character string for reviewers of the form First name Last name and multiple authors separated by a comma
-#' @param file_manager A character string for the name of the document maintainer of the form First name Last name
-#' @param revision A semantic version number of the form major.minor.patch. For development versions a fourth component is appended starting from .9000. The default is 0.0.0.9000 and should normally not be changed.
+#' @param reviewers A character string for reviewers of the form First name
+#' Last name and multiple authors separated by a comma
+#' @param file_manager A character string for the name of the document
+#' maintainer of the form First name Last name
+#' @param from_docx A character string with the path (absolute or relative) to
+#' a `.docx` file containing a pre-existing protocol.
+#' Please make sure to copy-paste all relevant meta-data from the `.docx` file
+#' to the corresponding parameters of this function.
+#' If nothing is provided, an empty template will be used.
+#' @param protocol_number A character string giving the protocol number.
+#' This parameter should normally not be specified (i.e. missing), unless
+#' `from_docx` is specified.
+#' A protocol number is a three digit string where the first digit corresponds
+#' with a theme and the last two digits identify a protocol within a theme.
```

Comment:
I think this is already clear from the help file (the explanations given in the arguments). The term `protocol code` is not exposed to the user. It is just something I used within the function body to refer to the combination of protocol type and number.

Reply:
Still, without an example of folder & filenames (or without trying out) I think the concrete idea of the folder structure and protocol filename syntax that will be created remains unclear to non-insiders. I believe the user does not exactly know how the result should look like.

Instead of an explicit protocol example, you could also refer to the INBO protocols repo as an example.

Reply:
See 
5c42d77